contracts/ForkDelta.sol

```diff
@@ -34,10 +33,9 @@ contract ForkDelta {
   }
 
   /// Constructor function. This is only called on contract creation.
-  function ForkDelta(address admin_, address feeAccount_, uint feeMake_, uint feeTake_, uint freeUntilDate_) public {
+  function ForkDelta(address admin_, address feeAccount_, uint feeTake_, uint freeUntilDate_) public {
```

Comment:
Here you change the function signature. Will this effect the API that interacts with the contract? 

Nevermind, this comment answers my question
"//... This is only called on contract creation."